modules/serverless-ossm-v1x-tls.adoc

```diff
@@ -0,0 +1,162 @@
+    -keyout custom.example.com.key \
+    -subj "/CN=custom-ksvc-domain.example.com/O=Example Inc."
+----
+. Sign the request with your CA:
++
+[source,terminal]
+----
+$ openssl x509 -req -days 365 -set_serial 0 \
+    -CA example.com.crt \
+    -CAkey example.com.key \
+    -in custom.example.com.csr \
+    -out custom.example.com.crt
+----
+. Check that the certificates appear in your directory:
++
+[source,terminal]
+----
+$ ls -1
+----
++
+.Example output
+[source,terminal]
+----
+custom.example.com.crt
+custom.example.com.csr
+custom.example.com.key
+example.com.crt
+example.com.key
+----
+. Create a secret:
++
+[source,terminal]
+----
+$ oc create -n istio-system secret tls custom.example.com \
+    --key=custom.example.com.key \
+    --cert=custom.example.com.crt
+----
+. Attach the secret to the Istio ingress gateway by editing the `ServiceMeshControlPlane` resource.
+.. Edit the `ServiceMeshControlPlane` resource:
++
+[source,terminal]
+----
+$ oc edit -n istio-system ServiceMeshControlPlane <control_plane_name>
+----
+.. Check that the following lines exist in the resource, and if they do not, add them:
++
+[source,yaml]
+----
+spec:
+  istio:
+    gateways:
+      istio-ingressgateway:
+        secretVolumes:
+        - mountPath: /custom.example.com
+          name: custom-example-com
+          secretName: custom.example.com
+----
+. Update the Istio ingress gateway to use your secret.
+.. Edit the `default-gateway` resource:
++
+[source,terminal]
+----
+$ oc edit gateway default-gateway
+----
+.. Check that the following lines exist in the resource, and if they do not, add them:
++
+[source,yaml]
+----
+- hosts:
+  - custom-ksvc-domain.example.com
+  port:
+    name: https
+    number: 443
+    protocol: HTTPS
+  tls:
+    mode: SIMPLE
+    privateKey: /custom.example.com/tls.key
+    serverCertificate: /custom.example.com/tls.crt
+----
+. Update the route to use pass-through TLS and `8443` as the `spec.port.targetPort`.
+.. Edit the route:
++
+[source,terminal]
+----
+$ oc edit route -n istio-system hello
+----
+.. Add the following configuration to the route:
++
+[source,yaml]
+----
+spec:
+  host: custom-ksvc-domain.example.com
+  port:
+    targetPort: 8443
+  tls:
+    insecureEdgeTerminationPolicy: None
+    termination: passthrough
+  to:
+    kind: Service
+    name: istio-ingressgateway
+    weight: 100
+  wildcardPolicy: None
+----
+
+.Verification steps
+
+* Access your serverless application by a secure connection that is now trusted by the CA:
++
+[source,terminal]
+----
+$ curl --cacert example.com.crt \
+    --header "Host: custom-ksvc-domain.example.com" \
+    --resolve "custom-ksvc-domain.example.com:443:<ingress_router_IP>" \
+     https://custom-ksvc-domain.example.com:443
+----
++
+[NOTE]
+====
+You must substitute your own value for `<ingress_router_IP>`.
+Steps for finding this IP or host name value varies depending on your {product-title} provider platform.
```

Comment:
I'm sorry, but my initial recommendation to change may vary -> varies here and below was wrong :(. Needs to be singular vary.